app/components/post_list/post_list.tsx

```diff
@@ -357,6 +395,29 @@ const PostList = ({
         return () => clearTimeout(t);
     }, [orderedPosts, highlightedId]);
 
+    const onScrollToEnd = () => {
+        scrollToIndex(0);
+    };
+
+    const ScrollToEndView = () => (
+        <Pressable
+            style={[
+                isNewMessages ? styles.scrollToEndBadge : styles.scrollToEndBtn,
+                {
+                    backgroundColor: isNewMessages ? theme.onlineIndicator : theme.sidebarBg,
+                },
+            ]}
+            onPress={onScrollToEnd}
+        >
+            <CompassIcon
+                size={16}
+                name='arrow-down'
+                color={theme.sidebarHeaderTextColor}
+            />
+            <Text>{isNewMessages && 'Jump To New Messages'}</Text>
```

Comment:
This needs to be translateable, also the check should be prior to mount the component instead of making the content optional.